Makefile.defs

```diff
@@ -32,15 +32,26 @@ CMAKE=cmake
 
 # set this to your ARM cross compiler:
 # set CC CXX AR LD RANLIB
-ifneq (,$(filter $(TARGET), kindle arm-generic))
-    export KINDLE=1
+ifeq ($(TARGET), arm-generic)
     CHOST?=arm-linux-gnueabi
-else ifeq ($(TARGET), kindle5)
+else ifeq ($(TARGET), kindle)
     export KINDLE=1
-    CHOST?=arm-kindle5-linux-gnueabi
+    HAS_KINDLE5_TC:=$(shell command -v arm-kindle5-linux-gnueabi-gcc 2> /dev/null)
+    ifdef HAS_KINDLE5_TC
+        #Use the custom build TC
+        CHOST?=arm-kindle5-linux-gnueabi
```

Comment:
Random note (this is fine): Using `:=` instead of `?:` would mean enforce using the custom TC in case something else is set in the env's CHOST.

I actually prefer always honoring the env, and this is what we previously did, so this is perfectly fine ;).